hack/util-install-bundle.sh

```diff
@@ -31,6 +31,19 @@ spec:
         dockerfile: true
         language: Go
         projectType: test-hermetic-build
+    - name: Hermetic build - python
+      pipelineParams:
+        - name: hermetic
+          value: "true"
+        - name: prefetch-input
+          value: "pip"
+      pipelineRef:
+        name: docker-build
+        bundle: ${BUNDLE}
+      when:
+        dockerfile: true
+        language: Python
```

Comment:
"python" is lowercase in https://github.com/cachito-testing/pip-e2e-test/blob/main/devfile.yaml#L3, not sure if that matters

Reply:
does not matter